src/systems/joint_controller/JointController.cc

```diff
@@ -164,6 +193,18 @@ void JointController::Configure(const Entity &_entity,
       return;
     }
   }
+  if ((!_sdf->HasElement("sub_topic")) && (!_sdf->HasElement("topic"))
+    && (this->dataPtr->useActuatorMsg))
+  {
+    topic = transport::TopicUtils::AsValidTopic("/actuators");
```

Comment:
So I think the logic is, if you have:
## **No** `<topic>` or `<sub_topic>` set 
#### With **no** `<use_actuator_msg>` set
It will listen to [**/model/*{model.name}*/joint/*`<joint_name>`*/cmd_vel**](https://github.com/gazebosim/gz-sim/blob/5d87ee2fb1156ee76436359f1adf9e95c3c3ad16/src/systems/joint_controller/JointController.cc#L182-L187) 
#### With `<use_actuator_msg>` set
It will listen to [**/actuators**](https://github.com/gazebosim/gz-sim/blob/5d87ee2fb1156ee76436359f1adf9e95c3c3ad16/src/systems/joint_controller/JointController.cc#L196-L199) 
(which is important for nested model control, like gimbals)
## `<sub_topic>` set
It will listen to [**/model/*{model.name}*/*`<sub_topic>`***](https://github.com/gazebosim/gz-sim/blob/5d87ee2fb1156ee76436359f1adf9e95c3c3ad16/src/systems/joint_controller/JointController.cc#L208-L212)
(overrides everything else except if `<topic>` is set and works with actuator or float msgs.)
## `<topic>` set
It will listen to [**/*`<topic>`***](https://github.com/gazebosim/gz-sim/blob/5d87ee2fb1156ee76436359f1adf9e95c3c3ad16/src/systems/joint_controller/JointController.cc#L224-L227)
(overrides everything else and works with actuator or float msgs.)

Reply:
Ah, thanks! I had to re-read the code. I was thinking `<topic>` wouldn't be used if `use_actuator_msgs=true` was set, but that's not the case.